lib/cog/command/pipeline/executor.ex

```diff
@@ -412,8 +416,8 @@ defmodule Cog.Command.Pipeline.Executor do
     end
```

Comment:
You might be able to avoid returning a function here, by changing the signature to `response_generator(room, message, adapter)` and using `&response_generator_fn(&1, message, adapter)` in all the places your currently using this.